browser/resources/settings/brave_overrides/site_details.ts

```diff
@@ -146,30 +146,6 @@ RegisterPolymerTemplateModifications({
     if (!usageSection) {
       console.error(`[Brave Settings Overrides] Couldn't find usageSection item`)
     } else {
-      usageSection.insertAdjacentHTML(
-        'afterend',
-        getTrustedHTML`
-          <div id="shields">
-            <div id="shieldsHeader"
-              style="padding: 0 var(--cr-section-padding);">
-              <h2 class="first"></h2>
-            </div>
-            <div class="list-frame">
-              <site-details-permission
-                category="[[contentSettingsTypesEnum_.BRAVE_SHIELDS]]">
-              </site-details-permission>
-            </div>
-          </div>
-        `)
-      const shieldsHeader =
-        templateContent.querySelector('div#shieldsHeader h2')
-      if (!shieldsHeader) {
-        console.error(
-          '[Brave Settings Overrides] Couldn\'t find Shields header')
-      } else {
-        shieldsHeader.textContent =
-            loadTimeData.getString('siteSettingsShields')
-      }
       const shieldsSettings = templateContent.querySelector(
```

Comment:
Do these Shields settings also need to be removed? I wasn't sure...?

Reply:
yes, looks working well without it, removed